client/views/add/add.js

```diff
@@ -17,6 +17,10 @@ function checkPrevMod(modBoxes) {
   return occurrences === 1;
 }
 
+function checkModsAccount(moderators) {
```

Comment:
@singhsanket143 is this empty function needed for anything?

Reply:
@peter-hank This was by mistake most probably. I have removed this.